asllib/Interpreter.mli

```diff
@@ -22,6 +22,7 @@
 module type S = sig
   module B : Backend.S
 
+  val run_env : (AST.identifier * B.value) list -> B.ast -> B.value B.m
```

Comment:
Can we put some documentation here? The name is not really explicit

Reply:
Of course, will do.